src/tasks/batch_integration/metrics/isolated_label_f1/script.py

```diff
@@ -13,16 +13,18 @@
 ## VIASH END
 
 print('Read input', flush=True)
-adata = ad.read_h5ad(par['input_integrated'])
+input_solution = ad.read_h5ad(par['input_solution'])
+input_integrated = ad.read_h5ad(par['input_integrated'])
 
+input_solution.obsp["connectivities"] = input_integrated.obsp["connectivities"]
+input_solution.obsp["distances"] = input_integrated.obsp["distances"]
 
-print('preprocess data', flush=True)
-adata.X = adata.layers['normalized']
-adata_int = adata.copy()
```

Comment:
ah, yes this shouldn't be necessary, since the transformers should be taking care of it 👍